src/soap.js

```diff
@@ -565,6 +565,22 @@ class SoapMethodCall {
         const that = this;
         const promise = this._transport(this.request);
         return promise.then(function(body) {
+            // Return a XML SOAP response in case of session expiration error
```

Comment:
I do not understand this. Shouldn't the session expiration have thrown an exception?

Reply:
Done thanks for the hint